test/e2e/proxy_test.go

```diff
@@ -177,9 +177,20 @@ func TestOAuthProxyE2E(t *testing.T) {
 		// },
 	}
 
-	image := os.Getenv("TEST_IMAGE")
 	backendImage := os.Getenv("HELLO_IMAGE")
 
+	// Get the image
+	// TODO: is there a better way?
+	alertmanagerPod, err := kubeClient.CoreV1().Pods("openshift-monitoring").Get(testCtx, "alertmanager-main-0", metav1.GetOptions{})
```

Comment:
maybe add a comment explaining why these values are used. I was unable to tell from the look of it.